### PR TITLE
Portfolio mobile menu

### PR DESCRIPTION
**Pull request by ntzwilly**
# Project 7: Mobile menu
## Project requirements
- Add the JavaScript linter.

- In mobile, implement the following features:

  - When the user clicks (or taps) the hamburger button, the mobile menu appears.
  
  - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
  
  - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -583,6 +591,91 @@ img {
...
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 44px;
+    text-align: left;
+    text-decoration: none;
+  }
+}
+
 @media screen and (min-width: 1024px) {
```
Comment (rloterh) [inline styles.css:679]: - [ ] Our design is appearing to be squashed when the breakpoint lower than **1024px**. 

Your design looks like this:
shot3

We want to ensure that our design fits the full width of the browser/ media screen at current whatever the breakpoint lesser than desktop screens. Let's ensure to improve upon this to make our design fit nicely on a specific media screen.

File: index.html [hunk 2]
```diff
@@ -23,24 +23,29 @@
...
+            ><img src="images/Logo-placeholder.svg" alt="logo"
+          /></a>
         </div>
-      </header>
+        <nav class="hamburger-container">
+          <div class="line1"></div>
+          <div class="line2"></div>
+          <div class="line3"></div>
+        </nav>
+        <nav class="header-nav-link">
```
Comment (rloterh) [inline index.html:38]: - [ ] Your implementation of the mobile menu is nice! However, we want both our design and features to follow the provided [Figma template]() as closely as possible. Your design is missing some hover effects on the mobile menu.

Here's your design when the menu is hovered:
shot2

Here's the original design for the same action:
shot4


Let's improve this by adding the hover effect to our menu items for mobile screen.

**Review by rloterh — CHANGES_REQUESTED** [review]
# Changes Required! ♻️

Hi Abraham and Willy 👋 👋, I'm Robert, your reviewer for this PR.

I must commend you for the nice work done👏.  You nailed almost all of the specifications of this project right. There some necessary changes we need to implement to improve this project.

## Good points 👍
No Linters errors ✔️
Good use of HTML semantics ✔️


## Aspects to improve ♻️

_Check the comment(s) under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const navDisplay = () => {
+  const burger = document.querySelector('.hamburger-container');
+  const nav = document.querySelector('.header-nav-link');
+  const main = document.querySelector('.main-background');
+  const logo = document.querySelector('.logo-image');
+
+  burger.addEventListener('click', () => {
```
[the remark this answers is not in the document]
Comment (ntzwilly) [reply script.js]: Thank you for your feedback. This issue is now fixed

File: styles.css (hunk 1 above)
Comment (ntzwilly) [reply styles.css:751]: I have made the required changes. The design fits now the full width

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**